<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCaliResultEditDialog</class>
 <widget class="QWidget" name="QCaliResultEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCaliResultEditDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="Select_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选中曲线</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="His_Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>历史曲线</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="Project_Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="K_Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Function_Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Methord_Edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="R0_Edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset_Button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>282</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_Button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_Button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
